index.html

```diff
@@ -9,6 +9,68 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Front-End Tools</h1> 
+  <div class="content">
+    <h1 class="title">Front-End Tools</h1>
+    <div class="grid-flex-display"> 
```

Comment:
I'd change this class name either to something simple like `.list` or something related to the content like `.tech-list`. The idea is to avoid attaching to a name like flex, grid or display which might be associated with css properties and we might use something different in the back.